src/main/java/com/github/princesslana/greedorama/Portfolio.java

```diff
@@ -18,6 +19,9 @@
 
   private final ImmutableList<Transaction> transactions;
 
+  public static final Ordering<Entry> byNetWorth =
```

Comment:
[nit] public static final variables BY_NET_WORTH

Reply:
Sorry, I'll fix that.